tests/integration/relation_tests/application-charm/metadata.yaml

```
@@ -0,0 +1,12 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+name: application
+description: |
+  Data platform libs application charm used in integration tests.
+summary: |
+  Data platform libs application meant to be used
+  only for testing of the libs in this repository.
+
+requires:
+  database:
+    interface: database-client
```

To match the other change, this should also be changed:

```suggestion
    interface: mongodb-client
```